tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+import os
+from allennlp.common.testing import AllenNlpTestCase
+AllenNlpTestCase.MODULE_ROOT = (os.path.dirname(os.path.abspath(__file__)) + '/../')
```

Comment:
The problem isn't due to python path, and `python -m pytest` doesn't solve the problem. Infact, root directory was already set in `pytest.ini` file as `python_paths = ./` 

This is due to `os.chdir()` usages. Eg.
```python 
# academic_paper_classifier_test.py
self.set_up_model('tests/fixtures/academic_paper_classifier.json',
                          'tests/fixtures/s2_papers.jsonl')
```
and below is relevant snippet of `model_test_case.py` in allennlp library
```python
class ModelTestCase(AllenNlpTestCase):
    def set_up_model(self, param_file, dataset_file):
        os.chdir(self.MODULE_ROOT)
```
But `self.MODULE_ROOT` is actually in pip installed library's allennlp module so it cant find `test` directory of current project. The best is to remove the `os.chdir(...)` in allennlp library itself. This is just temporary hack to work with `allennlp051`. Since we can't remove `os.chdir(...)` from already released `allennlp051`, this was just temporary hack.

Do you have any other suggestions to make this hacky solution clean?

Reply:
Also, if it isn’t clear: this hacky solution would get removed when the root issue is fixed and we upgrade this repo to the new version.